### DISCUSSION TITLE
研究證實聽笑話可以讓生活更美好

### DISCUSSION THREAD

**Discussion by Lipunpun** [1101-GS4719-C/Python-Community-GS4719-C#1161] [General]
最近在Dcard有看到類似的文章，無非是笑話可以刺激大腦，讓自己有愉悅感、並提升自身的創造力等等，為了讓大家在上最後一個小時的課能夠更撐下去，我來講個笑話：

一間餐廳裡面誰最厲害？
【顧客】(因為他們有點東西)

謝謝大家

**Comment by Brownnnn78** [comment]
接龍1
屈原的遺言是什麼?











:ㄅㄨ嚕ㄅㄨ嚕ㄅㄨ嚕

**Reply by HsuSungTing** [reply]
太地獄了我好喜歡

我也來補充一個
哈利波特裡面誰最有主見？

A : 佛地魔，因為他不會被牽著鼻子走

愚公臨死前召集兒子來到床邊虛弱的說：

「移山…移山………」

兒子：「亮晶晶？」

愚公卒。

**Reply by JohnnyXiao1024** [reply]
雖然大概會被判定內容不足
但我想要來講個爛笑話：

有一天橡皮筋起床 伸了個懶腰
.
.
然後他就彈出去了

**Comment by Lipunpun** [comment]
再跟大家分享一個新的笑話：

有一天牙籤A跟牙籤B在馬路上相遇，

突然一隻刺蝟快速的經過，

牙籤A就對牙籤B說 :

乾，我的公車過了...

很好笑吧，有沒有覺得身心靈昇華了一波。

**Comment by ziihui** [comment]
我也好喜歡廢笑話 那我也來分享一個
阿笠博士得老人痴呆會變什麼?



。
阿笠洗像

更

有天小華到天庭跟眾仙開會

小華 : 玉皇大帝，千手觀音為什麼還沒到呢 ?

玉皇大帝 : 因為她還在洗手...><

**Comment by Jamesmoneya** [comment]
我也來一個
劉備 字玄德

張飛 字翼德

五佰字？？

.

.

.

心得

**Comment by zilinng** [comment]
找笑話找到自己笑到停不下來

以下

醫生問小明：如果把你一邊耳朵割掉你會怎麼樣？
小明：我會聽不見
醫生又問：那再割掉另一個耳朵呢？
小明：我會看不見
醫生問他為什麼…
。
。
。
。
。
小明：因為我有戴眼鏡

**Comment by xiang305** [comment]
剛剛跟我朋友 a.k.a 笑話魔術師聊天
聽到一個還不錯的
想跟大家分享

為什麼狙擊手要躲在高處？
.
.
.
.
.
因為高端疫苗(易瞄)

**Comment by DuxtinTai** [comment]
適時的幽默真的很能放鬆生活中緊張步調
所以我也來分享一個

愚公臨死前召集兒子來到床邊虛弱的說：

「移山…移山………」

兒子：「亮晶晶？」
.
.
.
.
.
愚公卒。

**Comment by Lipunpun** [comment]
經過一個禮拜笑話的洗禮，大家有沒有覺得世界更美好了，經過這一個禮拜下來，大家的集思廣益，我覺得這就是人性溫暖的一面，希望大家更有勇氣可以面對春假結束後的期中考。

**Comment by lyien1021** [comment]
終於放假我也來分享

面試官 : 如果把自己比喻為一種動物，你會是哪一種?

小明 : 鸚鵡

面試官 : 喔? 為什麼是鸚鵡呢?

小明 : 喔? 為什麼是鸚鵡呢?

面試官 : ......
